src/core/qgsquick/qgsquickmapsettings.h

```diff
@@ -154,7 +154,7 @@ class QFIELD_CORE_EXPORT QgsQuickMapSettings : public QObject
     QgsRectangle extent() const;
 
     //! \copydoc QgsMapSettings::setExtent()
-    void setExtent( const QgsRectangle &extent, bool handleMargins = false );
+    Q_INVOKABLE void setExtent( const QgsRectangle &extent, bool handleMargins = false );
```

Comment:
Just wondering: property instead?

Reply:
@m-kuhn , the property is there, but it ignores the 2nd parameter (i.e. handleMargins), whereas I want our locators to not re-center onto a covered part of the canvas :)